gobblin-yarn/src/main/java/org/apache/gobblin/yarn/GobblinYarnConfigurationKeys.java

```diff
@@ -30,6 +30,9 @@ public class GobblinYarnConfigurationKeys {
   public static final String GOBBLIN_YARN_PREFIX = "gobblin.yarn.";
 
   // General Gobblin Yarn application configuration properties.
+  public static final String APP_MASTER_CLASS = GOBBLIN_YARN_PREFIX + "app.master.class";
+  public static final String DEFAULT_APP_MASTER_CLASS = GobblinApplicationMaster.class.getName();
```

Comment:
these are in the same package, so potentially OK... but just noting this introduces a circular dependency

Reply:
Worth calling out. I can't think of a scenario where this would be a problem but open to input